python/pyspark/pandas/namespace.py

```diff
@@ -2814,9 +2824,18 @@ def to_numeric(arg):
     1.0
     """
     if isinstance(arg, Series):
-        return arg._with_new_scol(arg.spark.column.cast("float"))
+        if errors == "coerce":
+            return arg._with_new_scol(arg.spark.column.cast("int"))
+        elif errors == "ignore":
+            scol = arg.spark.column
+            casted_scol = scol.cast("int")
+            return arg._with_new_scol(F.when(casted_scol.isNull(), scol).otherwise(casted_scol))
+        elif errors == "raise":
+            raise NotImplementedError("'raise' is not implemented yet, when the `arg` is Series.")
```

Comment:
Let's implement this case by using `assert_true` expression. e.g.) `assert_true(casted_col.isNotNull())`

Reply:
Sounds good. Let me try

Reply:
Btw, we should check the nullability of the original value as well; otherwise it also raises in the case `None` -> numeric.